src/encoded/visualization.py

```diff
@@ -814,12 +810,12 @@ def get_from_es(request, comp_id):
 
 def search_es(request, ids):
     '''Returns a list of composites from elastic-search, or None if not found.'''
-    key = "vis_composite"
+    index = "vis_composite"
     es = request.registry.get(ELASTIC_SEARCH, None)
-    if es and es.indices.exists(key):
+    if es and es.indices.exists(index):
         try:
             query = {"query": {"ids": {"values": ids}}}
-            res = es.search(body=query, index=key, doc_type='default', size=99999)  # size=200?
+            res = es.search(body=query, index=index, doc_type='default', size=99999)  # size=200?
             hits = res.get("hits", {}).get("hits", [])
             results = {}
             for hit in hits:
```

Comment:
I found significant and conflicting differences between master and the es5 branch in this file and shared_calculated_properties.py. I talked with @bek and he said that he was just taking master's versions, though it seems at least with visualizations.py we have differences. I didn’t look at it in detail because I’ve never looked at this code before.